src/check/test/type_checking_integration.zig

```diff
@@ -2177,7 +2177,6 @@ test "check type - comprehensive: polymorphism + lambdas + dispatch + annotation
         \\  where [
         \\    a.from_numeral : Numeral -> Try(a, [InvalidNumeral(Str)]),
         \\    b.from_numeral : Numeral -> Try(b, [InvalidNumeral(Str)]),
-        \\    b.from_numeral : Numeral -> Try(b, [InvalidNumeral(Str)]),
```

Comment:
Note that this was incorrect - it was a duplicate of the constraint right above it!